src/lectures/models/cmip.ipynb

```diff
@@ -0,0 +1,3361 @@
+{
```

Comment:
<p>Why are the absolute temperatures of the models so different? This could be confusing. Maybe plot the anomaly relative to a specific date?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/earth-env-data-science/earth-env-data-science-book/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>